iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -271,7 +269,7 @@ public override void UserEventTriggered(IChannelHandlerContext context, object @
         #endregion
 
         #region Connect
-        async void Connect(IChannelHandlerContext context)
+        async Task Connect(IChannelHandlerContext context)
         {
             if (Logging.IsEnabled) Logging.Enter(this, context.Name, nameof(Connect));
```

Comment:
Is it possible to capture this in E2E ?

Reply:
That is already captured in the existing E2E tests. All the mqtt-related E2E tests go through this code path.